server_extensions_extra/src/test/java/com/generalbytes/batm/server/extensions/extra/dash/sources/coinmarketcap/v2/CoinmarketcapV2APITest.java

```diff
@@ -0,0 +1,477 @@
+package com.generalbytes.batm.server.extensions.extra.dash.sources.coinmarketcap.v2;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import si.mazi.rescu.RestProxyFactory;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Class CoinmarketcapV2APITest tests method of interface ICoinmarketcapV2API.
+ */
+public class CoinmarketcapV2APITest {
+
+    private static long currentUnix = System.currentTimeMillis();
+
+    private static final long CACHE_EXPIRY_TIME_DEFAULT = 600;
+
+    private static ICoinmarketcapV2API api;
+
+    @BeforeClass
+    public static  void setup() {
+        api = RestProxyFactory.createProxy(ICoinmarketcapV2API.class, "https://api.coinmarketcap.com");
+    }
+
+    /**
+     * Method timeTest() tests if enauf time has expired and the listings should be updated.
+     *
+     * @throws InterruptedException
+     */
+    @Test
+    public void timeTest() throws InterruptedException {
+        Assert.assertTrue(true);
+        Thread.sleep(3000);
```

Comment:
No `sleep`s or any other waiting in the tests, please. This will make the build unnecessarily longer for everyone.

You can use `Calendar` to create unix time in the future.